bundles/org.openhab.binding.broadlinkthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,136 @@
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Floureon Thermostat Thing Type -->
+	<thing-type id="floureonthermostat">
+		<label>Floureon Thermostat</label>
+		<description>A heating device thermostat</description>
+
+		<channels>
+			<channel id="power" typeId="power"/>
+			<channel id="mode" typeId="mode"/>
+			<channel id="sensor" typeId="sensor"/>
+			<channel id="roomtemperature" typeId="roomtemperature"/>
+			<channel id="roomtemperatureexternalsensor" typeId="roomtemperatureexternalsensor"/>
+			<channel id="active" typeId="active"/>
+			<channel id="setpoint" typeId="setpoint"/>
+			<channel id="temperatureoffset" typeId="temperatureoffset"/>
+			<channel id="remotelock" typeId="remotelock"/>
+			<channel id="time" typeId="time"/>
+		</channels>
+
+		<representation-property>host</representation-property>
+
+		<config-description-ref uri="thing-type:broadlinkthermostat:floureonandhysenthermostat"/>
+	</thing-type>
+	<thing-type id="hysenthermostat">
+		<label>Hysen Thermostat</label>
+		<description>A heating device thermostat</description>
+
+		<channels>
+			<channel id="power" typeId="power"/>
+			<channel id="mode" typeId="mode"/>
+			<channel id="sensor" typeId="sensor"/>
+			<channel id="roomtemperature" typeId="roomtemperature"/>
+			<channel id="roomtemperatureexternalsensor" typeId="roomtemperatureexternalsensor"/>
+			<channel id="active" typeId="active"/>
+			<channel id="setpoint" typeId="setpoint"/>
+			<channel id="temperatureoffset" typeId="temperatureoffset"/>
+			<channel id="remotelock" typeId="remotelock"/>
+			<channel id="time" typeId="time"/>
+		</channels>
+
+		<representation-property>host</representation-property>
+
+		<config-description-ref uri="thing-type:broadlinkthermostat:floureonandhysenthermostat"/>
+	</thing-type>
+
+	<channel-type id="power">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Switch display on/off and enable/disables heating</description>
+		<category>Switch</category>
+	</channel-type>
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Mode</label>
+		<description>Current mode of the thermostat</description>
+		<state>
+			<options>
+				<option value="auto">auto</option>
+				<option value="manual">manual</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="sensor">
+		<item-type>String</item-type>
+		<label>Sensor</label>
+		<description>The sensor (internal/external) used for triggering the thermostat</description>
+		<category>Sensor</category>
+		<state>
+			<options>
+				<option value="internal">internal</option>
+				<option value="external">external</option>
+				<option value="internal_temp_external_limit">internal_temp_external_limit</option>
```

Comment:
The tag's value could be human readable.

Reply:
I changed it, but I even don't understand the meaning from the Thermostat's manual. ;)
https://www.ergo-tel.gr/manuals/HY03%20MANUAL%20PDF.pdf
0:internal sensor
1:external sensor
2:internal control temperature, external limit temperature